friends/src/component/FriendsList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "../App.css";
+import Card from "react-bootstrap/Card";
+
+const FriendsList = props => {
+  console.log(props.friends);
+
+  const friends = props.friends;
+
+  return (
+    <div className="friends-wrapper">
+      {friends.map(friend => (
+        <Card
+          className="m-5 mx-auto"
+          bg="dark"
+          text="white"
+          style={{ width: 400 }}
+          key={friend.id}
+        >
+          <Card.Body>
+            <Card.Title>{friend.name}</Card.Title>
```

Comment:
this is a good opportunity for object deconstruction 
```
const {name, age, email} = friend;
```
```suggestion
            <Card.Title>{name}</Card.Title>
```